pysal/contrib/viz/plot.py

```diff
@@ -9,38 +9,44 @@
 import pysal as ps
 import matplotlib.pyplot as plt
 
-
-def moran(var, w, xlabel='', ylabel='', title='', custom=(5,5)):
-	'''
+def mplot(m, xlabel='', ylabel='', title='', custom=(7,7)):
+    '''
     Produce basic Moran Plot 
     ...
-    Arguments
+    Parameters
     ---------
-    var	            : array
-                      values of variable
-    w       	    : spatial weight
+    m            : array
```

Comment:
Do I indicate it like this? Want to make sure I have it correctly so the
REST piece works later:

    m            : pysal.Moran object
                   output of pysal.esda.moran.Moran

On Mon, May 23, 2016 at 10:48 AM, Marynia Kolak <marynia.kolak@gmail.com>
wrote:

> But, that doesn't seem to be causing any real issues -- sorry to confuse!
> :P
>
> On Mon, May 23, 2016 at 10:39 AM, Marynia Kolak <marynia.kolak@gmail.com>
> wrote:
>
>> No worries, I know -- it was my mistake! I can update it later today.
>>
>> I was referencing a potential issue with how m is defined, and how the
>> class is being inherited (or not). I'm still getting familiar with under
>> the hood of PySal, and when I queried the structure of "m" :
>>
>> > type(m)
>> > instance
>>
>> The output was *instance*, which upon further reading, indicates that
>> the class might not be inheriting correctly. In other words, "type(m)"
>> should output the class object as a type, not "instance" (according to
>> those discussion and readings I linked earlier.
>>
>> On Mon, May 23, 2016 at 10:30 AM, Dani Arribas-Bel <
>> notifications@github.com> wrote:
>>
>>> In pysal/contrib/viz/plot.py
>>> <https://github.com/pysal/pysal/pull/797#discussion_r64241013>:
>>>
>>> >      ---------
>>> > -    var	            : array
>>> > -                      values of variable
>>> > -    w       	    : spatial weight
>>> > +    m            : array
>>>
>>> Not sure I understand your view. Mine was really that we need to give
>>> the user some guidance as to what to pass in as m for the function.
>>> Right now, it's asking for a (numpy) array, but what it really takes is
>>> the output of ps.Moran, hence my suggestion to use that in the
>>> documentation.
>>>
>>> —
>>> You are receiving this because you were mentioned.
>>> Reply to this email directly or view it on GitHub
>>> <https://github.com/pysal/pysal/pull/797/files/beace6676894fa61fd6a1328b8b0cc08e1fd8b49..6da47d169b5d14cf5d35592c41601653f4cc11b7#r64241013>
>>>
>>
>>
>